src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -33,34 +35,34 @@ import {CHANNEL_GROUP, GROUP_INFO, SIGNED} from '../../hooks/core/constant';
 import {COLORS} from '../../utils/theme';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {ProfileContact} from '../../components/Items';
-import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
+import useGroupInfo from '../GroupInfo/hooks/useGroupInfo';
 
 export const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
+  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: dimen.normalizeDimen(40)},
   users: {
-    paddingTop: 12
+    paddingTop: dimen.normalizeDimen(12)
   },
   photoProfile: {
-    height: normalize(50),
-    width: normalize(50)
+    height: dimen.normalizeDimen(50),
+    width: dimen.normalizeDimen(50)
   },
   btnAddText: {
     fontFamily: fonts.inter[600],
-    fontSize: normalizeFontSize(14),
+    fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(20),
-    color: COLORS.anon_primary
+    color: COLORS.signed_primary
   },
   btnAdd: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The normalization function `dimen.normalizeDimen` is used instead of `normalize` for styling. This change improves consistency in the codebase by using the same normalization function throughout.
```diff
- import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+ import {fonts, normalizeFontSize} from '../../utils/fonts';

- container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
+ container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: dimen.normalizeDimen(40)},

- photoProfile: {
-   height: normalize(50),
-   width: normalize(50)
- },
+ photoProfile: {
+   height: dimen.normalizeDimen(50),
+   width: dimen.normalizeDimen(50)
+ },

- btnAdd: {
-   padding: normalize(8),
-   backgroundColor: COLORS.lightgrey,
-   width: 'auto',
-   justifyContent: 'center',
-   alignItems: 'center',
-   position: 'absolute',
-   borderRadius: 8,
-   alignSelf: 'center',
-   bottom: 5
- },
+ btnAdd: {
+   padding: dimen.normalizeDimen(8),
+   backgroundColor: COLORS.lightgrey,
+   width: 'auto',
+   justifyContent: 'center',
+   alignItems: 'center',
+   borderRadius: 8,
+   alignSelf: 'center',
+   marginBottom: dimen.normalizeDimen(20)
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->